compiler-core/src/type_/error.rs

```diff
@@ -71,6 +71,47 @@ pub enum RecordVariants {
     NoVariants,
 }
 
+/// A suggestion for an unknown module
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
+pub enum ModuleSuggestion {
+    /// A module which which has a matching name, and an
+    /// exported value matching the one being accessed
+    Matching(EcoString),
+    /// A module already imported in the current scope
+    Imported(EcoString),
+    /// A module which can be imported
+    Importable(EcoString),
```

Comment:
What's the difference between importable and matching?

Reply:
Yeah, I was a little unsure how to implement this, because initially you suggested checking for a member with a name matching what was accessed. But then we also want to suggest in case the user made a typo. At the moment, `Matching` is a module with a matching name, and a matching export, whereas `Importable` is just any module which can be imported. `Matching` takes precedence over others, and  is the only one suggested in the code action. I don't really like how this works; do you have any suggestions as to how it should?

Reply:
If the module doesn't have the value being accessed then does it make sense to suggest importing it at all? It would be incorrect after the import still.

Reply:
That makes sense. I'll only suggest for typos or unimported modules with matching names. Thanks!